installer/macos/readme.txt

```diff
@@ -0,0 +1,13 @@
+Build Mac OS Disk Image
```

Comment:
This info should go into CONTRIBUTING.md then you can remove this file

Reply:
readme.txt moved to CONTRIBUTING.md and deleted